tests/tasks.py

```diff
@@ -293,3 +292,40 @@ def mytask(longer_arg):
             eq_(arg.names, ('longer-arg', 'l'))
             eq_(arg.attr_name, 'longer_arg')
             eq_(arg.name, 'longer_arg')
+
+        def decorated_function_names_are_preserved(self):
```

Comment:
I'm not sure how to run tests in this project. I eventually got frustrated and changed all the tests to `unittest.TestCase` and ran them using `nose`. How do you usually run tests?